content/en/observability_pipelines/troubleshooting.md

```diff
@@ -65,6 +65,14 @@ If you try to install a new version of the Worker in an instance that is running
 
 If you do not see Worker logs in [Log Explorer][12], make sure they are not getting excluded in your log pipelines. Worker logs must be indexed in Log Management for optimal functionality. The logs provide deployment information, such as Worker status, version, and any errors, that is shown in the Observability Pipelines UI. The logs are also helpful for troubleshooting Worker or pipelines issues. All Worker logs have the tag `source:op_worker`.
 
+## The Worker is not receiving logs from the source
+
+If you have to configure your source to send logs to the Worker, make sure the port that the Worker is listening on is the same port to which the source is sending logs.
```

Comment:
Should this be "If you have configured your source..." instead?

Reply:
Yes, thanks, updated!